### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adriel-tech**
Added broot to the file manager section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +125,7 @@ List of projects that provide terminal user interfaces
 <details open><summary><h2>File Managers</h2></summary>
 
 - [adbtuifm](https://github.com/darkhz/adbtuifm) A TUI file manager for Android, based on the Android Debug Bridge(ADB).
+- [broot](https://github.com/Canop/broot) A fuzzy finder file manager written in Rust.
```
Comment (ccoVeille) [inline README.md:128]: Is it the same broot available in miscellaneous section?

Comment (adriel-tech) [reply README.md:128]: Yes it is, I had tunnel vision while looking at the 'File Managers' section.

Comment (ccoVeille) [inline README.md:128]: Maybe the part about  tree navigation mentioned in previous entry could be kept here

**Review by adriel-tech — COMMENTED** [review]
Removed broot from misc and moved it to file manager using the original description.

**Review by ccoVeille — APPROVED** [review]

**Pull request merged**